source/opt/fold_spec_constant_op_and_composite_pass.h

```diff
@@ -35,22 +35,13 @@ class FoldSpecConstantOpAndCompositePass : public Pass {
   FoldSpecConstantOpAndCompositePass();
 
   const char* name() const override { return "fold-spec-const-op-composite"; }
-  Status Process(ir::Module* module) override {
-    Initialize(module);
-    return ProcessImpl(module);
-  };
+
+  Status Process(ir::Module* module) override;
```

Comment:
Right. I've moved some impl. code into `.cpp` so `<algorithm>` definitely need to be moved. For `<memory>`, we have `std::unique_ptr` in the header.